lib/sequel/adapters/shared/impala.rb

```diff
@@ -333,6 +333,16 @@ def metadata_dataset
         )
       end
 
+      # Impala doesn't like the word "integer"
+      def type_literal_generic_integer(column)
+        :int
+      end
+
+      # Impala doesn't like the word "biginteger"
+      def type_literal_generic_bignum(column)
```

Comment:
This should be `type_literal_generic_bignum_symbol`, at least when Sequel 4.36.0+ is being used.